frontend/src/App.tsx

```diff
@@ -233,9 +235,49 @@ function ThemeChangeButton() {
   );
 }
 
+function UpdatePopup() {
```

Comment:
Can this also go inside a folder with the other related module? Maybe `releases/UpdateSnack`?